### ISSUE TITLE
2020 final list

### ISSUE THREAD

**Issue by cjlortie** [cjlortie/Meta_contrasts#4]
1. Update writing.All me - need to add new 'results' that I completed and mention Stata too. I got this part totally.
2. Appendices.DONE a and b but..
a. Can you please check my file appendix.csv against the new manuals to ensure counts of objects, data etc is the same. I did this in 2019 and both were updated in 2020.
so, if you can save me some time and update appendix.csv and send back appendix_2020.csv to me updating IF anything has changed.  I did this by scraping the pdfs for each package but you can do by hand quick too.  
b. I used a classic Cochrane dataset and ran meta and metafor head to head in appendix b. Done.
However, I want to add a 'conclusion' section to the pdf (from an rmd).  Do you have the code handy to take outputs from models into tidy tables? I have done a few times but do not recall how? Tidymodels? I think?  Not sure if works for weird packages.
Basically, for appendix B I want to add table listing outputs from meta vs metafor nice and clean.
3. StataRun the Cochrane dataset in Stata, capture output to pdf, appendix c. DONEAre you willing to try this too?That is my goal for the day plus writing today. Then done.
